projects/libtiff/build.sh

```diff
@@ -15,33 +15,4 @@
 #
 ################################################################################
 
-
-# build zlib
-pushd "$SRC/zlib"
-./configure --static --prefix="$WORK"
-make -j$(nproc) CFLAGS="$CFLAGS -fPIC"
-make install
-popd
-
-# Build libjpeg-turbo
-pushd "$SRC/libjpeg-turbo"
-cmake . -DCMAKE_INSTALL_PREFIX=$WORK -DENABLE_STATIC=on -DENABLE_SHARED=off
-make -j$(nproc)
-make install
-popd
-
-cmake . -DCMAKE_INSTALL_PREFIX=$WORK -DBUILD_SHARED_LIBS=off
-make -j$(nproc)
-make install
-
-$CXX $CXXFLAGS -std=c++11 -I$WORK/include \
-    $SRC/tiff_read_rgba_fuzzer.cc -o $OUT/tiff_read_rgba_fuzzer \
-    -lFuzzingEngine $WORK/lib/libtiffxx.a $WORK/lib/libtiff.a $WORK/lib/libz.a $WORK/lib/libjpeg.a
-
-mkdir afl_testcases
-(cd afl_testcases; tar xf "$SRC/afl_testcases.tgz")
-mkdir tif
-find afl_testcases -type f -name '*.tif' -exec mv -n {} tif/ \;
-zip -rj tif.zip tif/
-cp tif.zip "$OUT/tiff_read_rgba_fuzzer_seed_corpus.zip"
-cp "$SRC/tiff.dict" "$OUT/tiff_read_rgba_fuzzer.dict"
+. contrib/oss-fuzz/build.sh
```

Comment:
looks strange to source the file. You probably meant ./contrib/oss-fuzz/build.sh ?